Source/WebCore/rendering/style/RenderStyleInlines.h

```diff
@@ -397,7 +397,7 @@ constexpr Style::ColumnCount RenderStyle::initialColumnCount() { return CSS::Key
 constexpr ColumnFill RenderStyle::initialColumnFill() { return ColumnFill::Balance; }
 inline Style::GapGutter RenderStyle::initialColumnGap() { return CSS::Keyword::Normal { }; }
 constexpr ColumnProgression RenderStyle::initialColumnProgression() { return ColumnProgression::Normal; }
-constexpr Style::LineWidth RenderStyle::initialColumnRuleWidth() { return CSS::Keyword::Medium { }; }
+constexpr Style::LineWidth RenderStyle::initialColumnRuleWidth() { return Style::LineWidth { 3.0f }; }
```

Comment:
This still seems like it will be wrong, since no zoom will be applied. I think you need to use a "style-builder-custom": "Initial" for these properties and and ensure the zoom is applied.